src/components/Tool/FixPhoneNumbers/FixPhoneNumbers.tsx

```diff
@@ -207,25 +196,9 @@ const FixPhoneNumbers: React.FC<Props> = ({
     handleDeleteModalClose();
   };
 
-  const handleAdd = (personId: string, number: string): void => {
+  const handleChange = (personId: string, numbers: PhoneNumberData[]): void => {
```

Comment:
It looks like an invalid phone number can't be deleted. I think it should be able to be deleted.